src/plugins/workspace/public/components/workspace_detail_app.tsx

```diff
@@ -105,6 +106,10 @@ export const WorkspaceDetailApp = (props: WorkspaceDetailProps) => {
   const handleWorkspaceFormSubmit = useCallback(
     async (data: WorkspaceFormSubmitData) => {
       let result;
+      if (isFormSubmitting) {
+        return;
+      }
+      setIsFormSubmitting(true);
```

Comment:
I think it's better to move `setIsFormSubmitting(true);` after this block:
```
if (!currentWorkspace) {
    notifications?.toasts.addDanger({
      title: i18n.translate('Cannot find current workspace', {
        defaultMessage: 'Cannot update workspace',
      }),
    });
    return;
  }
```
Because it has an early `return`, we should set the submitting state after this `return`

Reply:
updated